src/dll/edbgEcmdDll.C

```diff
@@ -1685,54 +1689,8 @@ uint32_t dllQueryHostMemInfoRanges( const std::vector<ecmdChipTarget> & i_target
 
 #ifdef EDBG_ISTEP_CTRL_FUNCTIONS
 
-enum IplMode_t
```

Comment:
These (and the rest of the changes from here) seem unrelated to the commit description. Are they required? If so please split these into a separate commit. Thanks!

Reply:
Updated commit message accordingly. Removing old code with ipl_init() coming to set the istep attribute

Reply:
This was not resolved, if the code is required please split into a separate commit. Thanks.